### DISCUSSION TITLE
Welcome to Composed Discussions!

### DISCUSSION THREAD

**Discussion by shaps80** [composed-swift/Composed#33] [Ideas]
@JosephDuffy I'll let you start this off if you have any initial ideas about how best to structure this. I was thinking it'd be a great place to kick off proposals, directions, etc...?

**Comment by JosephDuffy** [comment]
https://github.com/composed-swift/Composed/issues/15 can probably be moved to a discussion (I don't see the option for that but maybe it's a permission issue).

https://github.com/composed-swift/Composed/blob/master/.github/ISSUE_TEMPLATE/proposal.md could also be replaced with a new discussions category?

I don't see any other things that we have right now that could benefit from discussions, but it should definitely be the starting point for future discussions that aren't issues with the existing code.
